src/main/java/apoc/schema/Schemas.java

```diff
@@ -417,6 +435,20 @@ private IndexConstraintNodeInfo nodeInfoFromIndexDefinition(IndexReference index
         List<String> properties = new ArrayList<>();
         Arrays.stream(indexReference.properties()).forEach((i) -> properties.add(tokens.propertyKeyGetName(i)));
         try {
+            float populationProgressValue;
+            try {
+                final PopulationProgress populationProgress = schemaRead.indexGetPopulationProgress(indexReference);
+                populationProgressValue = populationProgress.getCompleted() / populationProgress.getTotal() * 100;
+            } catch (IncompatibleClassChangeError e) {
+                // Workaround because in Neo4j 3.5.19 we have the following error
+                // Failed to invoke procedure `apoc.schema.nodes`: Caused by: java.lang.IncompatibleClassChangeError: Found interface org.neo4j.storageengine.api.schema.PopulationProgress, but class was expected
+                populationProgressValue = db.execute("CALL db.indexes() YIELD tokenNames, properties, progress " +
```

Comment:
@conker84 I think you said it will work across all versions if we use the procedure approach? If so, let's do that and not bother calling the API in the lines above.